src/analyses/call_graph.cpp

```diff
@@ -96,6 +80,53 @@ void call_grapht::add(
 
 /*******************************************************************\
 
+Function: call_grapht::compute_reachable
+
+  Inputs:
+
+ Outputs:
+
+ Purpose:
+
+\*******************************************************************/
+
+void call_grapht::compute_reachable(
```

Comment:
There is goto-programs/compute_called_functions.cpp - the code of which will also take into account functions the address of which is taken.